libpod/options.go

```diff
@@ -241,6 +242,20 @@ func WithCNIPluginDir(dir string) RuntimeOption {
 	}
 }
 
+func WithImageRuntime(config storage.StoreOptions) RuntimeOption {
```

Comment:
Instead of this, can we initialize it in NewRuntime?

Reply:
The store should always be the same one as the Runtime store, so we can just pass the store from the Runtime into the ImageRuntime

Reply:
fixed.